src/webextension/list/manage/reducers.js

```diff
@@ -7,6 +7,18 @@ import { combineReducers } from "redux";
 import * as actions from "../actions";
 import { cacheReducer, listReducer } from "../reducers";
 
+export function accountReducer(state = { mode: "guest" }, action) {
```

Comment:
Will we want this in the doorhanger as well? I notice below that the "account_status_updated" message is handled in `list/message-ports.js`, which applies to both the manager and the doorhanger, but the doorhanger won't have a reducer to respond to that action.

Reply:
None of the visual prototypes have included this kind of information.

I can move the reducers and actions into `manage/`.

Reply:
We can see what - if anything - Ryan has to say about this, but if it's not necessary, then the only thing we'd maybe want to do is have the message port listener in the doorhanger ignore the account update event. Even then, we could just have it fire and be a no-op...

Reply:
word from @changecourse is that there are "no plans \[to include account information on the doorhanger]".

I'll shift things around so that `list/popup` doesn't have to see any of this.

Reply:
Ok, so shifting all the account-details stuff "down" the tree requires a lot more changes than shifting the reducer up, and ignoring it.

I'm going to go with that, since it puts the actions and reducers at the same level, and is already ignored by things that don't care about it.

Reply:
Will we want to be able to respond to FxA account events in the doorhanger? (e.g. if something caused us to be logged out, should we respond by locking the content of the doorhanger?) If so, we'd probably want to have the account reducer in the doorhanger.